src/components/devices/mqtt/bat.vue

```diff
@@ -26,8 +27,8 @@
 							componentId
 						}}/get/imported</openwb-base-copy-to-clipboard
 					><br />
-					Geladene Energie in Wh, mit Nachkommastellen (Float), nur
-					positiv<br />
+					Geladene Energie in Wh, Float mit Punkt als
```

Comment:
Die Begriffe "Float" und "Integer" sind schon sehr spezifisch und werden nicht von jedem verstanden. Ich hatte das daher nur in Klammern als Zusatzinfo angegeben.
Alternativvorschlag:
"Geladene Energie in Wh, Zahl mit oder ohne Nachkommastellen (Float, Integer) und einem Punkt als Dezimaltrenner, nur positiv."